adapter/package.json

```diff
@@ -29,7 +28,7 @@
     "peerDependencies": {
         "@dhis2/app-runtime": "^2.0.4",
         "@dhis2/d2-i18n": "^1.0.5",
-        "@dhis2/ui-core": "^4.6.1",
+        "@dhis2/ui": "^5.0.2",
```

Comment:
We had `ui-widgets` as a dependency, and `ui-core` as a peerDependency. `ui` is now a peerDependency. Not quite sure if that is correct because I don't really know the rationale behind them being separated like that before.